intro-HPC/ch_introduction.tex

```diff
@@ -272,8 +315,20 @@ \subsection{What programming languages can I use?}
 \operatingsystem. Supported and common programming languages on the \hpc are
 C/C++, FORTRAN, Java, Perl, Python, MATLAB, R, etc.
 
-Supported and commonly used compilers are GCC, clang, J2EE and Intel Cluster
-Studio.
+Supported and commonly used compilers are 
+%TODO insert compilers for other sites
+\ifantwerpen
+GCC, clang, J2EE and Intel Cluster Studio.
+\fi
+\ifleuven
+GCC, Intel and PGI. 
+\fi
+\ifbrussel
+GCC, clang, J2EE and Intel Cluster Studio.
+\fi
+\ifgent
+GCC, clang, J2EE and Intel Cluster Studio.
```

Comment:
Since nobody complained before I made changes ONLY for Leuven...

From: Kenneth Hoste [mailto:notifications@github.com]
Sent: donderdag 18 februari 2016 14:05
To: hpcugent/vsc_user_docs <vsc_user_docs@noreply.github.com>
Cc: Mag Selwa <mag.selwa@kuleuven.be>
Subject: Re: [vsc_user_docs] Feature/leuven chapter1 (#106)


In intro-HPC/ch_introduction.tex<https://github.com/hpcugent/vsc_user_docs/pull/106#discussion_r53310412>:

> @@ -272,8 +315,20 @@ \subsection{What programming languages can I use?}

>  \operatingsystem. Supported and common programming languages on the \hpc are

>  C/C++, FORTRAN, Java, Perl, Python, MATLAB, R, etc.

>

> -Supported and commonly used compilers are GCC, clang, J2EE and Intel Cluster

> -Studio.

> +Supported and commonly used compilers are

> +%TODO insert compilers for other sites

> +\ifantwerpen

> +GCC, clang, J2EE and Intel Cluster Studio.

> +\fi

> +\ifleuven

> +GCC, Intel and PGI.

> +\fi

> +\ifbrussel

> +GCC, clang, J2EE and Intel Cluster Studio.

> +\fi

> +\ifgent

> +GCC, clang, J2EE and Intel Cluster Studio.

(for Ghent, at least)

—
Reply to this email directly or view it on GitHub<https://github.com/hpcugent/vsc_user_docs/pull/106/files#r53310412>.